programs/scope/src/handlers/handler_initialize_tokens_metadata.rs

```diff
@@ -0,0 +1,24 @@
+use anchor_lang::prelude::*;
+
+#[derive(Accounts)]
+#[instruction(feed_name: String)]
+pub struct InitializeTokensMetadata<'info> {
+    #[account(mut)]
+    pub admin: Signer<'info>,
+
+    #[account(seeds = [b"conf", feed_name.as_bytes()], bump, has_one = admin)]
+    pub configuration: AccountLoader<'info, crate::Configuration>,
+
+    #[account(init, payer = admin, space = 8 + std::mem::size_of::<crate::TokenMetadatas>())]
+    pub token_metadatas: AccountLoader<'info, crate::TokenMetadatas>,
```

Comment:
Shouldn't this be an account zero (already allocated)? I think it is too big for allocation via cpi.

```suggestion
    #[account(zero)]
    pub token_metadatas: AccountLoader<'info, crate::TokenMetadatas>,
```